modules/claims_api/config/routes.rb

```diff
@@ -8,6 +8,7 @@
     namespace :forms do
       ## 526 Forms
       post '526', to: 'disability_compensation#submit_form_526'
+      post '526/:id/upload_documents', to: 'disability_compensation#upload_supporting_documents'
```

Comment:
I'm thinking form/526/:id/attachments might be a cleaner end point for this from a documentation standpoint but I could be wrong.

Reply:
Yeah you right... will refactor with better nomenclature